cmd/k8s-bigip-ctlr/main.go

```diff
@@ -369,6 +359,77 @@ func verifyArgs() error {
 	return nil
 }
 
+func getCredentials() error {
+	if len(*credsDir) > 0 {
+		var usr, pass, bigipURL string
+		var usrBytes, passBytes, urlBytes []byte
+		var err error
+		if strings.HasSuffix(*credsDir, "/") {
+			usr = *credsDir + "username"
```

Comment:
This won't work with the .spec.volumes[].secret.items key and path functionality. If they place the files deeper into the directory structure or change the name we'll fail. I'm not suggesting we support this but we should call this out as a warning in the docs to NOT do that.